### PR TITLE
add Darwin support for readline

### PR DESCRIPTION
**Pull request by alcharkov**

### CODE REVIEW

File: install-sdk [hunk 1]
```diff
@@ -32,11 +32,12 @@ prereqs() {
         hash pip3 2>/dev/null || { missing=1; echo >&2 "Please install pip3, which is available through your package manager or https://pip.pypa.io/en/latest/installing.html"; }
         hash sdl-config 2>/dev/null || { missing=1; echo >&2 "Please install SDL (and its headers), which is available through your package manager or http://www.libsdl.org/"; }
     fi
-    [ -e /usr/include/readline/readline.h ] || { missing=1; echo >&2 "Please install readline (and its headers), which is available through your package manager (libreadline-dev on Debian/Ubuntu)"; }
     if [[ $unamestr == 'Darwin' ]]; then
        [ -e "/usr/local/Cellar/boost/"*"/include/boost/config.hpp" ] || { missing=1; echo >&2 "Please install boost (and its headers) via Homebrew"; }
+       [ -e /usr/local/include/readline/readline.h ] || { missing=1; echo >&2 "Please install readline (and its headers), which is available through your package manager (libreadline-dev on Debian/Ubuntu)"; }
```
Comment (ddevault) [inline install-sdk]: Since this branch is only taken for Darwin it's probably worth clarifying which package it is on homebrew rather than debian

Comment (alcharkov) [reply install-sdk]: Hi, it came from homebrew

Comment (ddevault) [reply install-sdk]: Re-read this text.

Comment (alcharkov) [reply install-sdk]: Hi, sorry, did the changes as requested.

**Comment by ddevault** [comment]
Thanks!

**Comment by MaxLeiter** [comment]
On macOS 10.14.3 and with readline 8.0, I get:
```bash
Please install readline (and its headers) via Homebrew
```
I installed readline via homebrew. Not doing anything as root.

**Pull request merged**